flaml/automl/task/generic_task.py

```diff
@@ -3,7 +3,7 @@
 from typing import List, Optional
 
 import numpy as np
-
+from sklearn.utils import shuffle
```

Comment:
It has been imported in original line 41.

Reply:
ah ic..
I will close the PR :)
Thank you for letting me know...